.gitignore

```diff
@@ -1,5 +1,6 @@
 # Editor temporary/working/backup files #
 #########################################
+env/
```

Comment:
🤔

Reply:
Oops yeah that wasn't supposed to be committed